src/routers/VaultItemPage.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import sampleData from "../config/data/sampleData";
+
+function VaultItemPage() {
+  const { collectionsId, vaultId } = useParams();
```

Comment:
I think `vaultId` could become `vaultItemId`, since it's the ID of the VaultItem